app/views/partnering_organizations/subscriptions.html.erb

```diff
@@ -0,0 +1,3 @@
+
+<%= react_component("partnering_organizations/PartneringOrgSubscriptions",
+  {subscribers: @subscribers}) %>
```

Comment:
We can just directly do `current_partnering_organization.subscriptions` here (yours works too but just needs that extra method on the controller up there. I think just passing in the params is stylistically better here)